py/desispec/quicklook/quicklook.py

```diff
@@ -406,7 +406,8 @@ def setup_pipeline(config):
 
     psf=None
     if config["Flavor"] == 'arcs':
-        os.mkdir(os.path.join(os.environ['QL_SPEC_REDUX'],'calib2d','psf',config["Night"]))
+        if not os.path.exists(os.path.join(os.environ['QL_SPEC_REDUX'],'calib2d','psf',config["Night"])):
+            os.mkdir(os.path.join(os.environ['QL_SPEC_REDUX'],'calib2d','psf',config["Night"]))
```

Comment:
Why do we need this here? findfile should handle this.

Reply:
This is needed to create an output directory for desi_bootcalib. Yes, findfile will find the psfboot file for other steps of the pipeline, but it is needed so that desi_bootcalib will run since we are currently running this step using the runcmd function. This will be updated once we come to a conclusion on where QL output should go for arcs, but for now this keeps the user from having to create a calib2d/psf/night directory before running QL on arcs.